contracts/interfaces/IAssessment.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+interface IAssessment {
+
+  /* ========== DATA STRUCTURES ========== */
+
+  enum PollStatus { PENDING, ACCEPTED, DENIED }
+
+  enum EventType { CLAIM, INCIDENT }
+
+  enum Asset { ETH, DAI }
+
+  enum UintParams {
+    MIN_VOTING_PERIOD_DAYS,
+    MAX_VOTING_PERIOD_DAYS,
+    PAYOUT_COOLDOWN_DAYS,
+    REWARD_PERC,
+    INCIDENT_IMPACT_ESTIMATE_PERC,
+    CLAIM_ASSESSMENT_DEPOSIT_PERC,
+    INCIDENT_ASSESSMENT_DEPOSIT_PERC
+  }
+
+  struct Configuration {
+    // The minimum number of days the users can vote on polls
+    uint8 MIN_VOTING_PERIOD_DAYS;
+    // The maximum number of days the users can vote on polls
+    uint8 MAX_VOTING_PERIOD_DAYS;
+    // Number of days the users must wait after a poll closes, to redeem payouts.
+    uint8 PAYOUT_COOLDOWN_DAYS;
+    // Percentage used to calculate assessment rewards (0-10000 i.e. double decimal precision)
+    uint16 REWARD_PERC;
+    // Percentage used to calculate potential impact of an incident
+    uint16 INCIDENT_IMPACT_ESTIMATE_PERC;
```

Comment:
I'd drop "estimate" in the name, it's a bit misleading. 

incident_burn_perc?
incident_payout_perc?
max_payout_perc?

Reply:
It's a generic estimate used for all incidents (probably 30%). Wouldn't dropping the "estimate" lead you o think that it's value depends on each specific incident rather than applying the same percentage to all?

Reply:
> Wouldn't dropping the "estimate" lead you o think that it's value depends on each specific incident rather than applying the same percentage to all?

not really. estimate makes me think it's something variable, not a "flat" number

Reply:
An "estimate" means we use some formula to estimate it. We're using a percentage to determine it and it's definitely not an estimate but a decision to burn that amount.

Reply:
incident_flat_impact_perc?